bundle/ios/GutenbergNativeTranslations.swift

```diff
@@ -3,9 +3,16 @@ import Foundation
 private func dummy() {
     _ = NSLocalizedString("'%s' isn't yet supported on WordPress for Android", comment: "")
     _ = NSLocalizedString("'%s' isn't yet supported on WordPress for iOS", comment: "")
+    _ = NSLocalizedString("(555)555-1234", comment: "translators: sample content for \"Contact\" page template")
```

Comment:
Is it expected that we don't see similar changes for Android?

Reply:
They're there in [bundle/android/strings.xml](https://github.com/wordpress-mobile/gutenberg-mobile/pull/1987/files#diff-8a38584e71e09430592cd316ae4d6851), but GitHub has the bad habit of not showing some diffs by default, so it was easy to miss